web/message_bar.css

```diff
@@ -0,0 +1,153 @@
+	@media screen and (forced-colors: active) {
+		--message-bar-close-button-color: ButtonText;
+		--message-bar-close-button-border: 1px solid ButtonText;
+		--message-bar-close-button-hover-bg-color: ButtonText;
+		--message-bar-close-button-active-bg-color: ButtonText;
+		--message-bar-close-button-focus-bg-color: ButtonText;
+		--message-bar-close-button-color-hover: HighlightText;
+	}
+
+	display: flex;
+	position: relative;
+	padding: 12px 8px 12px 0;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	gap: 8px;
+	align-self: stretch;
+
+	border-radius: 4px;
+
+	border: 1px solid var(--message-bar-border-color);
+	background: var(--message-bar-bg-color);
+	color: var(--message-bar-fg-color);
+
+	> div {
+		display: flex;
+		padding-inline-start: 16px;
+		align-items: flex-start;
+		gap: 8px;
+		align-self: stretch;
+		padding-inline-end: 32px;
+
+		&::before {
+			content: "";
+			display: inline-block;
+			width: 16px;
+			height: 16px;
+			mask-image: var(--message-bar-icon);
+			mask-size: cover;
+			background-color: var(--message-bar-icon-color);
+		}
+
+		> div {
+			display: flex;
+			flex-direction: column;
+			align-items: flex-start;
+			gap: 8px;
+			flex: 1 0 0;
+
+			.title {
+				font-size: 13px;
+				font-weight: 590;
+			}
+
+			.description {
+				font-size: 13px;
+			}
+		}
+	}
+
+	.closeButton {
+		position: absolute;
+		width: 32px;
+		height: 32px;
+		inset-inline-end: 8px;
+		inset-block-start: 8px;
+		background: none;
+		border-radius: var(--message-bar-close-button-border-radius);
+		border: var(--message-bar-close-button-border);
+
+		&::before {
+			content: "";
+			display: inline-block;
+			width: 16px;
+			height: 16px;
+			mask-image: var(--closing-button-icon);
+			mask-size: cover;
+			background-color: var(--message-bar-close-button-color);
+		}
+
+		&:is(:hover, :active, :focus)::before {
+			background-color: var(--message-bar-close-button-color-hover);
+		}
+
+		&:hover {
+			background-color: var(--message-bar-close-button-hover-bg-color);
+		}
+
+		&:active {
+			background-color: var(--message-bar-close-button-active-bg-color);
+		}
+
+		&:focus {
+			background-color: var(--message-bar-close-button-focus-bg-color);
+		}
+
+		> span {
+			display: inline-block;
+			width: 0;
+			height: 0;
+			overflow: hidden;
+		}
+	}
+}
+
+#annotationRemovedToast {
+  --message-bar-icon: url(images/secondaryToolbarButton-documentProperties.svg);
+  --message-bar-icon-color: #0060DF;
+	--message-bar-bg-color: #deeafc;
+	--message-bar-fg-color: #15141a;
+	--text-primary-color: #15141a;
+	--message-bar-border-color: rgb(0 0 0 / 0.08);
+
+	.undoButton {
+		background: rgba(21, 20, 26, 0.14);
+		border-radius: 4px;
+		font-weight: bold;
+		color: #15141a;
+		border: none;
+    padding: 8px 16px;
+	}
+
+  > div {
+    align-items: center;
+  }
+
+  .closeButton {
+    padding-top: 12px;
+  }
+}
+
+#toastContainer {
+	position: fixed;
+	bottom: 100px;
+	left: 50%;
+	transform: translateX(-50%);
+}
```

Comment:
Did you test that everything is rendering fine when switching to HCM mode on Windows ?

Reply:
Sorry I didn't respond to this comment earlier, it has indeed been tested on Win11 and it works as expected.